### PR TITLE
Redirect to Stripe KYC to complete progressive onboarding to enable payouts

### LINKED ISSUE #4812: Progressive onboarding prototype – Update overview page
### Description
Update the overview page to allow the merchant to finish the onboarding redirecting them to Stripe.

### Acceptance criteria
- Adding a button or link which redirects to Stripe KYC only if payments or payouts are disabled and the account is using progressive onboarding.

### PR DESCRIPTION
**Pull request by oaratovskyi**
Fixes #4812 

#### Changes proposed in this Pull Request

Tweak the task list (Things To Do) by supporting completing the second step of KYC to enable payouts for merchants.

#### Testing instructions

* Checkout to server branch `dev/ask-for-eventually-due`.
* Complete the progressive onboarding, check the Stripe account that it has payments enabled and payouts disabled.
* Find `Update WooCommerce Payments business details` block and click `Update` button that should redirect to the Stripe KYC asking for business details and bank account.
* Verify that it allowed payouts on the same Stripe account (need to wait a few mins)

#### Screenshots
image

image

image
image
image

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by oaratovskyi** [comment]
JFYI I did not add the changelog, since it'll be merged into the prototype branch, not into develop.

File: includes/wc-payment-api/class-wc-payments-api-client.php [hunk 1]
```diff
@@ -1182,25 +1182,31 @@ public function request_capability( string $capability_id, bool $requested ) {
...
+				'create_live_account'  => ! $this->is_in_dev_mode(),
+				'actioned_notes'       => $actioned_notes,
+				'progressive'          => ! empty( $account_data ),
+				'progressive_complete' => $progressive_complete,
+				'account_data'         => $account_data,
 			]
 		);
 
+		if ( $progressive_complete ) {
+			unset( $request_args['force_on_boarding'] );
```
Comment (ismaeldcom) [inline includes/wc-payment-api/class-wc-payments-api-client.php]: I wonder if that's just for testing purposes 🤔

**Review by ismaeldcom — COMMENTED** [review]
Thanks for taking care of it! I'm not sure if we should keep it like this for the initial prototype or go further.

We are rendering the button always, and not only when it's required. And as a followup we should probably look at the task `Update WooCommerce Payments business details` that's pretty much the same.

File: includes/wc-payment-api/class-wc-payments-api-client.php (hunk 1 above)
Comment (oaratovskyi) [reply includes/wc-payment-api/class-wc-payments-api-client.php]: oh, yeah, I'll remove it 🙈 good catch!

**Comment by oaratovskyi** [comment]
@ismaeldcom I agree that we could re-use existing task, I did that for the demo and was a bit in hurry, but now I guess it should be made right 😅

File: includes/wc-payment-api/class-wc-payments-api-client.php (hunk 1 above)
Comment (oaratovskyi) [reply includes/wc-payment-api/class-wc-payments-api-client.php]: Done in https://github.com/Automattic/woocommerce-payments/pull/4848/commits/c8d1dd310c2042c8e0ac947cb7c8bccf95cd41e2

**Review by ismaeldcom — APPROVED** [review]
Thanks for the update. Now it meets the issue requirements and looks quite integrated with our current flow 😄

**Comment by oaratovskyi** [comment]
No need to check post-merge checkboxes, it will be shipped when `prototype/progressive-onboarding` is merged

**Pull request merged**